src/ripple/protocol/jss.h

```diff
@@ -597,6 +597,9 @@ JSS(warnings);                // out: server_info, server_state
 JSS(workers);
 JSS(write_load);   // out: GetCounts
 JSS(NegativeUNL);  // out: ValidatorList; ledger type
+JSS(NotValidAfter);
+JSS(NotValidBefore);
```

Comment:
This file has historically ordered the JSS items alphabetically, with first letter upper case items up top. All three of these should be moved to fit in with that order.